deploy/14-deploy-eBTC-oracle.ts

```diff
@@ -14,11 +21,11 @@ const func: DeployFunction = async ({ getNamedAccounts, deployments, network }:
 
   eBTC_Accountant = eBTC_Accountant || (await ethers.getContract("MockAccountant_eBTC")).address;
 
-  const SNAPSHOT_UPDATE_INTERVAL = ethers.constants.MaxUint256;
-  const eBTC_ANNUAL_GROWTH_RATE = ethers.utils.parseUnits("0.15", 18);
+  const eBTC_ANNUAL_GROWTH_RATE = SECONDS_PER_YEAR; // 0% growth rate
```

Comment:
Clarification: Chaos Labs is suggesting setting an annual growth rate of 0% ([here](https://community.venus.io/t/provide-support-for-capped-oracles-for-enhanced-security/5092/6)), but the CorrelatedTokenOracle doesn't support an annual growth rate equal to 0, so we are setting the minimum acceptable value for the annual growth rate. This value is almost negligible, so we should be safe